src/coreclr/jit/hwintrinsic.cpp

```diff
@@ -34,6 +34,19 @@ static const HWIntrinsicInfo hwIntrinsicInfoArray[] = {
         /* category */ category \
     },
 #include "hwintrinsiclistarm64.h"
+#elif defined (TARGET_RISCV64)
+#define HARDWARE_INTRINSIC(isa, name, size, numarg, t1, t2, t3, t4, t5, t6, t7, t8, t9, t10, category, flag) \
```

Comment:
nit: maybe better to define this macro once under `#if defined(TARGET_XARCH) || defined (TARGET_ARM64) || defined (TARGET_RISCV64)`.